src/RedisOptions.php

```diff
@@ -261,4 +260,26 @@ public function getPassword()
     {
         return $this->getResourceManager()->getPassword($this->getResourceId());
     }
+
+    /**
+     * Set resource user
+     *
+     * @param string $user ACL User
+     * @return RedisOptions Provides a fluent interface
+     */
+    public function setUser($user)
```

Comment:
Since this is a new method, we can add argument type here.
```suggestion
    public function setUser(string $user): void
```

I'd also remove `return $this;` so we can introduce `: void` return type.

Reply:
I added the argument type, but kept the RedisOptions return type to be compatible with the other setters and getters

Reply:
Fine for me, will delete them in v4 anyways.